zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/UnassignUserFromGroupTest.java

```diff
@@ -58,7 +58,8 @@ void initClientAndInstances() {
             .send()
             .join()
             .getGroupKey();
-    client.newAssignUserToGroupCommand(groupKey).userKey(userKey).send().join();
+    // TODO: this will be changed with unassign user command refactor
```

Comment:
🙃 Would you mind adding an issue number so we can ensure it isn't missed. Unless you're already coding this and have a PR almost ready for this of course.